recipe/meta.yaml

```diff
@@ -27,14 +35,18 @@ outputs:
     build:
       entry_points:
         - f2py = numpy.f2py.f2py2e:main  # [win]
+      ignore_run_exports:
+        - libgfortran5  # [osx]
+      missing_dso_whitelist:
+        - '$RPATH/ld64.so.1'  # [s390x]
     requirements:
       build:
         - {{ compiler("c") }}
-        # libllvm10 10.0.0 build 1 has an issue building numpy, use build 0
-        - libllvm10 10.0.0 h21ff451_0  # [osx]
+        - {{ compiler("fortran") }}
       host:
         - python
         - pip
+        - setuptools <=51  # [win]
```

Comment:
What's about setuptools <49.2.0 for all platforms, see https://github.com/numpy/numpy/blob/68752f786df542d340f25c41a8920d9b2aed66cf/pyproject.toml#L4
Is there any specific on win?